user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Configuring_components/Customize_Component_Layout.md

```diff
@@ -28,7 +28,17 @@ Each component in a dashboard or low-code app has a number of default options. B
 
      - In the *Colors* section, specify a custom background color and/or font color, either by specifying the color in RGB format or by using the color picker box on the right.
 
-       From DataMiner 10.0.12 onwards, under *Colors* > *Color palette*, you can customize additional component colors, e.g. for the lines in a line chart.
+       Under *Colors* > *Data colors* or *Colors* > *Color palette* (prior to DataMiner 10.3.0 [CU16]/10.4.0 [CU4]/10.4.7<!--RN 39739-->), you can customize additional component colors, e.g. for the lines in a line chart.
```

Comment:
Just a headsup, this structure changes again in https://intranet.skyline.be/DataMiner/Lists/Release%20Notes/DispForm2.aspx?ID=39746: 
![image](https://github.com/SkylineCommunications/dataminer-docs/assets/113689648/f532e435-694f-472c-9d83-f9ae4a636533)